python/pyarrow/tests/test_convert_builtin.py

```diff
@@ -995,6 +994,9 @@ def test_sequence_timestamp():
     'ns'
 ])
 def test_sequence_timestamp_with_timezone(timezone, unit):
+    pytest.importorskip("pytz")
+    import pytz
```

Comment:
I think you can simplify this to a single line as `pytz = pytest.importorskip("pytz")`

(and the same for all other occurrences)

Reply:
This one is not yet addressed I think